pages/design/rule-design.md

```diff
@@ -18,19 +18,43 @@ The WCAG-ACT-RULES-CG rule design builds on WCAG 2.x and its supporting document
 
 ## Rule Properties
 
-### Rule name and identifier
+The rule file should have a unique hyphenated filename, created from condensing the name and the unique identifier.
 
-The rule must have a unique name, preferably a two or three word topic, as well as an identifier. This uses the following format: **SC#-#-#-identifier**
+### Rule identifier
 
-- **SC#-#-#**: This is an identifier for the criterion to which the test case applies. #-#-# stands for the number of that criterion, such as SC4-1-2.
+The 6-digit identifier is created using a random hex generator, such as https://www.browserling.com/tools/random-hex. The identifier remains the same if the rule name changes.
 
-- **+SC#-#-#**: This can be used if the test case applies to multiple success criteria, such as SC1-1-1+SC4-1-2-identifier. The numbers are in the same order as they are used in WCAG.
+### Rule name
 
-- **identifier**: This must be a lower case identifier of the test, preferably no more then 3 words. It can only contain alphanumeric values or a dash (-).
+The agreed conventions for naming of rules are as follows:
 
-### Test aspects
+- Be succinct, direct and declarative, and avoid unnecessary words, eg. an, the.
+- Use declarative verbs, eg. has, have, is.
+- Use back ticks to indicate words from code, eg. label v. Label.
+- Identify code as an element or attribute, eg. label element.
+- Use sentence case, eg. Heading is descriptive, unless using code, eg. id attribute value is unique, or initials, eg. HTML.
+- Use singular tense so that necessary plurals stand out, eg. video element has captions.
+- Front load the applicable thing, eg. Button has accessible name.
+- Refer to each applicable thing in a consistent way across rules, especially when related or checking the same applicable thing.
+- Describe the passing condition, eg. id value is unique, rather than page has no duplicate ids.
+- Do not use hyphens or dashes unless correct for code, eg. aria-*.
+- Do not use camelCase unless correct for code, eg. iframe not iFrame, feImage in SVG.
+- If unsure, refer to WCAG language used.
+- Aim to be unique, which should happen if title includes the applicable thing and passing condition.
 
-A test aspect is a part of a test subject that must be available in order to properly run the test. The ACT Rules Format defines the following:
+### Rule type
+
+The rule type is either atomic or composite.
+
```

Comment:
I think we should add something about rule description.
For example: full sentence (ended by a full period), starts with "This rule checks that …", declarative rather than negative.

Reply:
I'm not sure why it's not there. I thought I made changes to the list at the top of the page also.